.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,53 @@
+name: Unit tests
+
+on:
+  push:
+    branches:
+      - master
+      - next
+  pull_request:
+    branches:
+      - master
+      - next
+
+jobs:
+  test-node:
+    name: Test on Node.js v${{ matrix.node-version }} and Jest v${{ matrix.jest-version }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [10.x, 12.x, 13.x, 14.x, 16.x]
```

Comment:
13 is EOL, but since it was tested on travis let's keep it

(10 is EOL as well, but that's documented in README as minimum version)